javascript-carousel/main.js

```diff
@@ -0,0 +1,481 @@
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.remove('hidden');
+  jigglypuffDiv.classList.add('hidden');
+};
+function handleSquirtleDot5() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'jigglypuff';
+  pikachuDiv.classList.add('hidden');
+  bulbasaurDiv.classList.add('hidden');
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.add('hidden');
+  jigglypuffDiv.classList.remove('hidden');
+};
+
+function handleJigglypuffLeftAngle() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'squirtle';
+  pikachuDiv.classList.add('hidden');
+  bulbasaurDiv.classList.add('hidden');
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.remove('hidden');
+  jigglypuffDiv.classList.add('hidden');
+};
+function handleJigglypuffRightAngle() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'pikachu';
+  pikachuDiv.classList.remove('hidden');
+  bulbasaurDiv.classList.add('hidden');
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.add('hidden');
+  jigglypuffDiv.classList.add('hidden');
+};
+function handleJigglypuffDot1() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'pikachu';
+  pikachuDiv.classList.remove('hidden');
+  bulbasaurDiv.classList.add('hidden');
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.add('hidden');
+  jigglypuffDiv.classList.add('hidden');
+};
+function handleJigglypuffDot2() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'bulbasaur';
+  pikachuDiv.classList.add('hidden');
+  bulbasaurDiv.classList.remove('hidden');
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.add('hidden');
+  jigglypuffDiv.classList.add('hidden');
+};
+function handleJigglypuffDot3() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'charmander';
+  pikachuDiv.classList.add('hidden');
+  bulbasaurDiv.classList.add('hidden');
+  charmanderDiv.classList.remove('hidden');
+  squirtleDiv.classList.add('hidden');
+  jigglypuffDiv.classList.add('hidden');
+};
+function handleJigglypuffDot4() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'squirtle';
+  pikachuDiv.classList.add('hidden');
+  bulbasaurDiv.classList.add('hidden');
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.remove('hidden');
+  jigglypuffDiv.classList.add('hidden');
+};
+function handleJigglypuffDot5() {
+  clearInterval(timer);
+  timer = setInterval(handleSetInterval, '3000');
+  currentScreen = 'jigglypuff';
+  pikachuDiv.classList.add('hidden');
+  bulbasaurDiv.classList.add('hidden');
+  charmanderDiv.classList.add('hidden');
+  squirtleDiv.classList.add('hidden');
+  jigglypuffDiv.classList.remove('hidden');
+};
```

Comment:
This is a _really_ brute-force way to approach this. I'd try to make this happen with single, shared `handleDot`, `handleRightAngle` and `handleLeftAngle` event listeners that use `currentScreen` to examine which image is showing and `event.target` to figure out which one was clicked.